hazelcast/src/main/java/com/hazelcast/cp/internal/MembershipChangeSchedule.java

```diff
@@ -163,11 +165,12 @@ public void readData(ObjectDataInput in) throws IOException {
             groupId = in.readObject();
             membersCommitIndex = in.readLong();
             int len = in.readInt();
-            members = new HashSet<>(len);
+            Set<RaftEndpoint> membersSet = new LinkedHashSet<>(len);
```

Comment:
can you please add a simple note in the code for preference of LinkedHashSet over HashSet?

Reply:
Thanks, Ahmet, for the comment!
Have changed the implementation from `HashSet` to `LinkedHashSet`, because the actual type of `members` in the `CPGroupMembershipChange` constructor is `LinkedHashSet`. 
So when we restore the serialized `CPGroupMembershipChange` object with `HashSet`, the order of `members` has changed.
I will add this note to the code.